src/org/sparkboard/slack/account_bridge_node.cljs

```diff
@@ -0,0 +1,54 @@
+(ns org.sparkboard.slack.account-bridge-node
+  (:require [org.sparkboard.firebase.tokens :as tokens]
+            [kitchen-async.promise :as p]
+            [org.sparkboard.slack.slack-db :as slack-db]
+            [org.sparkboard.http :as http]
+            [applied-science.js-interop :as j]))
+
+;; this code runs in the legacy node server and handles inbound links (from slack to sparkboard).
+;;
+;; Slack sign-in status is authoritative - if the user has linked accounts,
+;; we automatically sign them in to Sparkboard when they land here.
```

Comment:
That probably won’t happen until quite a bit later (until the main legacy server is also migrated) because linking accounts relies on re-using a user’s Sparkboard session, which is restricted by subdomain (so we can’t read it by directing users to Eg jvm.sparkboard.com, if they are signed in to demo.sparkboard.com)